python/guide/module/gluon/normalization.md

```diff
@@ -0,0 +1,412 @@
+# Normalization Blocks
+
+When training deep neural networks there are a number of techniques that are thought to be essential for model convergence. One important area is deciding how to initialize the parameters of the network. Using techniques such as Xavier initialization, we can can improve the gradient flow through the network at the start of training. Another important technique is normalization: i.e. scaling and shifting certain values towards a distribution with a mean of 0 (i.e. zero centered) and a standard distribution of 1 (i.e. unit variance). Which values you normalize depends on the exact method used as we'll see later on.
+
+<p align="center">
+    <img src="./imgs/data_normalization.jpeg" alt="drawing" width="500"/>
+    <p align="center">Figure 1: Data Normalization
+        <a href="http://cs231n.github.io/neural-networks-2/">(Source)</a>
+    </p>
+</p>
+
+Why does this help? [Some research](https://papers.nips.cc/paper/7515-how-does-batch-normalization-help-optimization.pdf) has found that networks with normalization have a loss function that's easier to optimize using stochastic gradient descent. Other reasons are that it prevents saturation of activations and prevents certain features from dominating due to differences in scale.
+
+### Data Normalization
+
+One of the first applications of normalization is on the input data to the network. Although this whole section can be skipped in favor of an initial `BatchNorm` layer in your network, which will have a very similar effect (see next section).
+
+* **Step 1** is to calculate the mean and standard deviation of the entire training dataset. You'll usually want to do this for each channel separately. Sometimes you'll see normalization on images applied per pixel, but per channel is more common.
+* **Step 2** is to use these statistics to normalize each batch for training and for inference too.
+
+Warning: You should calculate the normalization means and standard deviations using the training dataset only. Any leakage of information from you testing dataset will effect the reliability of your testing metrics.
+
+When using pre-trained models from the Gluon Model Zoo you'll sometimes see the normalization statistics used for training. You'll want to use these statistics to normalize your own input data for fine-tuning or inference with these models. Using `transforms.Normalize` is one way of applying the normalization, and this should be used in the `Dataset`.
+
+
+```python
+import mxnet as mx
+from mxnet.gluon.data.vision.transforms import Normalize
+
+image_int = mx.nd.random.randint(low=0, high=256, shape=(1,3,2,2))
+image_float = image_int.astype('float32')/255
+normalizer = Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
+image = normalizer(image_float)
+image
+```
+
+
+
+
+    
+    [[[[-1.31304061  1.17004883]
```

Comment:
This will be generated and can be removed.

Reply:
updated.